WordPress/wordpress-playground
# wp-now: Remove `.wp-env.json` support

Fixes https://github.com/WordPress/wordpress-playground/issues/405

## What?

Removes support for `.wp-env.json` and `.wp-env.override.json`.

## Why?

Our support for the 'core' argument is woefully incomplete (https://github.com/WordPress/wordpress-playground/issues/405#issuecomment-1557189257). Rather than try to sort through all of those permutations right now, it's easier to just remove support.

## How?

Removes a bunch of code.

## Testing Instructions

1. Tests should pass.
2. Run `git clone git@github.com:WordPress/gutenberg.git` and `nx preview wp-now start --path=/path/to/gutenberg-clone` and verify `wp-now` starts as expected.
